docs/appendices/release-notes/5.6.4.rst

```diff
@@ -47,4 +47,5 @@ See the :ref:`version_5.6.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Fixed an issue that caused ``LIKE`` operator matching an empty string pattern
+  in ``WHERE`` clause to return invalid results.
```

Comment:
I'd add add an example to make it clear for users.

Reply:
updated, Thanks